cmd/tendermint/commands/lite.go

```diff
@@ -66,17 +65,21 @@ func runProxy(cmd *cobra.Command, args []string) error {
 	}
 
 	// First, connect a client
+	logger.Info("Connecting to source HTTP client...")
 	node := rpcclient.NewHTTP(nodeAddr, "/websocket")
 
-	cert, err := proxy.GetCertifier(chainID, home, nodeAddr)
+	logger.Info("Constructing certifier...")
+	cert, err := proxy.NewCertifier(chainID, home, node, logger)
 	if err != nil {
-		return err
+		return cmn.ErrorWrap(err, "constructing certifier")
 	}
+	cert.SetLogger(logger)
```

Comment:
Is this redundant? We are passing logger to a constructor at line 72 above and set it in the constructor body?